omod/src/main/java/org/openmrs/module/htmlformentryui/fragment/controller/htmlform/EnterHtmlFormFragmentController.java

```diff
@@ -118,7 +120,7 @@ public void controller(FragmentConfiguration config, UiSessionContext sessionCon
 				hf = htmlFormEntryService.getHtmlFormByForm(form);
 			} else if (StringUtils.isNotBlank(definitionUiResource)) {
 				hf = HtmlFormUtil.getHtmlFormFromUiResource(resourceFactory, formService, htmlFormEntryService,
-				    definitionUiResource, encounter);
```

Comment:
Did you intentionally remove `encounter`?

Reply:
I added it back.